src/main/java/frc/robot/maps/TempestMap.java

```diff
@@ -1,32 +1,121 @@
+                return EncodedSpeedController.join(motors,new MockEncoder());
+
+
+            }
 
             @Override
             public EncodedSpeedController getRight() {
-                final SendableSpeedController rightGroup = SendableSpeedController.group(new WPI_TalonSRX(2),
-                        new WPI_TalonSRX(3));
-                return EncodedSpeedController.join(rightGroup, new MockEncoder());
+                SendableSpeedController motors = SendableSpeedController
+                        .wrap(new SpeedControllerGroup(new WPI_TalonSRX(2), new WPI_TalonSRX(3)));
+
+                return EncodedSpeedController.join(motors,
+                        new MockEncoder());
             }
+        };
+    }
+
+    @Override
+    public IndexMap getIndexerMap() {
+        return new IndexMap() {
 
             @Override
-            public EncodedSpeedController getLeft() {
-                final SendableSpeedController leftGroup = SendableSpeedController.group(new WPI_TalonSRX(1),
-                        new WPI_TalonSRX(4));
-                return EncodedSpeedController.join(leftGroup, new MockEncoder());
+            public SendableSpeedController singulator() {
+                final Victor indexingMotor = new Victor(4);
+                return SendableSpeedController.wrap(indexingMotor);
+            }
+
+            @Override
+            public SendableSpeedController pierreMotor() {
+                final Victor pierreMotor = new Victor(5);
+                return SendableSpeedController.wrap(pierreMotor);
+            }
+            
+            public BooleanSupplier frontIntakeIR() {
+                AnalogTrigger frontIntakeIR = new AnalogTrigger(0);
+                frontIntakeIR.setLimitsVoltage(1.2,1.4);
+                return frontIntakeIR::getTriggerState;
             }
+
+            public BooleanSupplier bottomPierreIR() {
+                AnalogTrigger bottomPierreIR = new AnalogTrigger(1);
+                bottomPierreIR.setLimitsVoltage(1.2, 1.4);
+                return bottomPierreIR::getTriggerState;            }
+
+            public BooleanSupplier topPierreIR() {
+                AnalogTrigger topPierreIR = new AnalogTrigger(2);
+                topPierreIR.setLimitsVoltage(1.2, 1.4);
+                return topPierreIR::getTriggerState;            }
+
+            public BooleanSupplier backIntakeIR() {
+                AnalogTrigger backIntakeIR = new AnalogTrigger(3);
+                backIntakeIR.setLimitsVoltage(1.2, 1.4);
+                return backIntakeIR::getTriggerState;
+            }
+
+
+    
+            // AnalogTrigger frontIntakeIR = new AnalogTrigger(0);
+            // AnalogTrigger bottomPierreIR = new AnalogTrigger(1);
+            // AnalogTrigger topPierreIR = new AnalogTrigger(2);
+            // AnalogTrigger backIntakeIR = new AnalogTrigger(3);
+
+        };
+    }
+
+    @Override
+    public ShooterMap getShooterMap() {
```

Comment:
Again, not sure why these are showing up here, but there shouldn't be shooter stuff in this PR.